contrib/flare/template.py

```diff
@@ -0,0 +1,51 @@
+pkgname = "flare"
+pkgver = "0.15.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "blueprint-compiler",
+    "cargo-auditable",
+    "pkgconf",
+    "protoc",
+    "meson",
+    "gettext",
+    "desktop-file-utils",
+]
+makedepends = [
+    "cairo-devel",
+    "gdk-pixbuf-devel",
+    "glib-devel",
+    "gtk4-devel",
+    "gtksourceview-devel",
+    "libadwaita-devel",
+    "pango-devel",
+    "libspelling-devel",
+]
+pkgdesc = "Unofficial Signal Desktop app"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "AGPL-3.0-only"
+url = "https://mobile.schmidhuberj.de/flare"
+source = f"https://gitlab.com/schmiddi-on-mobile/flare/-/archive/{pkgver}/flare-{pkgver}.tar.gz"
```

Comment:
It would be possible to get a tarball that already includes vendored rust libs, but the URL is https://gitlab.com/schmiddi-on-mobile/flare/-/package_files/139350251/download, which obviously doesn't work as it would need a new url on each new version.

If there's anything I'm missing wrt getting the "package" from https://gitlab.com/schmiddi-on-mobile/flare/-/releases/0.15.0 which includes the vendored source code, I'd appreciate a pointer there as well.

Reply:
there's no issue with running cargo vendor so there's not much point

Reply:
yep, with your suggestion it works as it should and without the hacky make_dir changes. Thanks!